### ISSUE TITLE
Bug in SessionPerRequestModule

### ISSUE THREAD

**Issue by rpmcnally** [codai/Sharp-Lite#6]
I believe there might be an issue with the SessionPerRequestModule, but I'm not completely sure.  Sharp Lite has been working fine for months, but after updating to the latest version (from about a month ago) I started getting the following exception:

Event handlers can only be bound to HttpApplication events during IHttpModule initialization.

It is complaining about the Dispose method in SessionPerRequestModule.  This post seems to indicate that code is unecessary:

http://stackoverflow.com/questions/6877045/event-handlers-can-only-be-bound-to-httpapplication-events-during-ihttpmodule-in

I know that particular code didn't change so I'm not sure why I didn't experience it before.  I also tried rolling back the Sharp Lite DLLs but am still consistently getting the exception.

-Patrick

**Comment by rpmcnally** [comment]
I may have figured out why I wasn't encountering the error before.  I believe the error has been happening all along but Visual Studio just wasn't showing it.  Something is wrong with the 'SharpLite.*.pdb' files in the templify packages.  The pdb files created in the lib folder by templify do not match the pdb files in Template/TemplateSrc/lib even though the dlls do.

I used chkmatch to try and verify the pdb files against the dlls.  The pdb files in Template/TemplateSrc/lib do match the dlls.  When I ran it with the pdb files in the templified directory, chkmatch said the pdb file was an 'unknown format'.  My guess is that without the correct debugging information (e.g. pdb file), Visual Studio did not show the exception since it doesn't happen during the request lifecycle.
